notebooks/instructlab-knowledge/instructlab-knowledge.ipynb

```diff
@@ -727,21 +710,21 @@
    "source": [
     "# Summary\n",
     "\n",
-    "To recap, given a source document in PDF format, this notebook:\n",
+    "To recap, given a source documents in PDF format, this notebook:\n",
```

Comment:
Are you thinking of a specific tense for this, because before the tenses varied a lot and were inconsistent. It does matter to me which one, It should all just be in the same tense

Reply:
What would you suggest the tense we should stick with? Maybe we do a follow up PR aligning all of the tenses in all of the documentation within each section?

CC: @JustinXHale